imageprocessor/imageprocessor-core/src/main/java/pl/edu/uj/kimage/Image.java

```diff
@@ -1,5 +1,5 @@
 package pl.edu.uj.kimage;
```

Comment:
It seems to me that we already have discussed commit messages. This message says nothing. I add this to the Contributor Guide.